sign.example.json

```diff
@@ -24,8 +24,12 @@
     }
   },
   "cookies": [
+	
     "mi18nLang=en-us; ltoken=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa; ltuid=123456789;",
-    "mi18nLang=en-us; ltoken=bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb; ltuid=234567890;"
+    "mi18nLang=en-us; ltoken=bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb; ltuid=234567890;",
+	"mi18nLang=en-us; ltoken_v2=v2_ccccccccccccc; ltuid_v2=345678901; ltmid_v2=abcdefghijklm;"
```

Comment:
Use space, match real case number of letter
leave one v1 and v2 example each is ok